tools/devtools/src/page_scripts/owl_devtools_global_hook.js

```diff
@@ -1484,7 +1485,9 @@
         return;
       }
       if (objectType === "subscription") {
-        owl.reactive(obj)[key] = value;
+        window?.owl?.reactive
+          ? owl.reactive(obj)[key]
+          : (window.__OWL_DEVTOOLS__.reactive(obj)[key] = value);
```

Comment:
these parens are not placed correctly, this problem would go away if you extracted reactive and toRaw once during construction.